src/api/entities/Asset/NonFungible/NftCollection.ts

```diff
@@ -164,7 +165,12 @@ export class NftCollection extends BaseAsset {
         }
 
         const details = await neededMetadata.details();
-        return { ...details, id, type, ticker };
+
+        if (type === MetadataType.Local) {
+          return { ...details, id, type, ticker };
+        } else {
+          return { ...details, id, type };
```

Comment:
I include the ticker here so the object can uniquely identify local keys, but it felt confusing/sloppy to include it in for global where it isn't relevant.